src/redux/store.js

```diff
@@ -0,0 +1,9 @@
+import {configureStore} from '@reduxjs/toolkit';
+
+import reducer from './reducers';
```

Comment:
do we want to keep all reducers in one file in the future?

Reply:
Each reducer has its own separate file (we have one reducer so far - `authReducer`)  and we combine them in `redux/reducers/index.js`.